src/KKManager/ModpackTool/Data/ModpackToolConfiguration.cs

```diff
@@ -15,32 +15,44 @@ namespace KKManager.ModpackTool
 {
     public class ModpackToolConfiguration : IXmlSerializable, INotifyPropertyChanged
     {
-        private static ModpackToolConfiguration _instance;
+        private static ModpackToolConfiguration s_instance;
```

Comment:
Code style used in the project starts all private fields with a `_`. `s_` is not used, instead _ + title case is used.